packages/jest-config/src/validConfig.js

```diff
@@ -57,6 +57,9 @@ module.exports = ({
   noStackTrace: false,
   notify: false,
   preset: 'react-native',
+  reporters: [
```

Comment:
@thymikee Would appreciate some feedback from you on this. There was `ValidationError` being thrown but after a certain number of tries when I added a sample reporters file here, it stopped throwing the error. Shouldn't adding it to `types/Config.js` be suffice